jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/skaffold/FilesTaskV2.java

```diff
@@ -139,10 +139,7 @@ private void addGradleFiles(Project project) {
     skaffoldFilesOutput.addBuild(project.getBuildFile().toPath());
 
     // Add settings.gradle
-    if (project.getGradle().getStartParameter().getSettingsFile() != null) {
-      skaffoldFilesOutput.addBuild(
-          project.getGradle().getStartParameter().getSettingsFile().toPath());
-    } else if (Files.exists(projectPath.resolve(Settings.DEFAULT_SETTINGS_FILE))) {
+    if (Files.exists(projectPath.resolve(Settings.DEFAULT_SETTINGS_FILE))) {
```

Comment:
I think there is some loss in fidelity here, we cannot handle the case where there is a custom settings file. We could just inject this into the task?

Reply:
I'm not sure how to handle this, the option has been deprecated in Gradle and will be removed. See: https://docs.gradle.org/8.4/userguide/upgrading_version_8.html#configuring_custom_build_layout

Reply:
Ah I see. Okay this works.